frontend/package.json

```diff
@@ -115,10 +116,6 @@
     "ts-node": "^10.5.0",
     "typescript": "^4.5.5"
   },
-  "resolutions": {
```

Comment:
Did you remove this intentionally?

Reply:
Yes, I wanted to ask you about that, since it caused a problem in the install stage, when i invoked yarn with `--frozen-lockfile`:
```
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
```
Does these packages need to have fixed versions?

Reply:
That's an unrelated error. Yes those versions should be fixed. I'd suggest you
- add them back
- run `yarn` without `--frozen-lockfile`
- commit the changes to yarn.lock

Reply:
The thing is, that yarn.lock won't change if i run it without `--frozen-lockfile`. I tried it, and run `yarn --ignore-platform --frozen-lockfile` afterwards again and the Problem remains. Or do you suggest, that `--frozen-lockfile` should never be used?

Reply:
I got the idea from this: https://github.com/yarnpkg/yarn/issues/4778#issuecomment-339448000

Reply:
We're only using `--frozen-lockfile` in CI, so we can make sure that the dependencies installed in an automated CI build are exactly the ones we expect

Reply:
That was my intention. You can try to run `yarn --ignore-platform --frozen-lockfile` on the `develop` branch. That's where i am getting the error too. For some reason the `Test Frontend`-Workflow does not use the frozen-flag. Why is that so?

Reply:
Out of curiosity: what makes these Package version so important to stay at that fixed version?

Reply:
@Kadrian I kept the resolutions and adjusted the version of babel-loader in the dependencies to be the same as in the resolutions. This works for me now.